src/SDKs/Azure.Core/data-plane/Azure.Core/Net/ClientPipeline.cs

```diff
@@ -13,7 +13,7 @@ public struct HttpPipeline
     {
         static readonly PipelineTransport s_defaultTransport = new HttpPipelineTransport();
         static readonly PipelinePolicy s_defaultLoggingPolicy = new LoggingPolicy();
-        // TODO (pri 2): what are the default status codes to retry?
+        // TODO (pri 1): I am not sure this should be here. Maybe we need retry policy per service
```

Comment:
Interesting. As part of the API review and those processes, aren't the return codes something that could get standardized accross Azure services?

Reply:
Yeah, it would be great if whole Azure standardized here, but I assume it's difficult technically and organizationally. @johanste might have more insight here.

Reply:
The status codes are defined in (RFC7231)[https://tools.ietf.org/html/rfc7231#section-6]. 

Generally speaking, if the response has a Retry-After (or a x-ms-retry-after) header, it is retryable. Most common status codes for this are 429 and 5xx. In fact, 429s are often retried by our libraries regardless of if the response has a retry-after header after a suitable delay (30s). Services are being *strongly* encouraged to include the Retry-After header. 

What we don't have is a set of common (error codes)[https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses]. Which we really should create an inventory of.